Backend/src/app.ts

```diff
@@ -46,6 +46,7 @@ class App {
       api: { version },
     } = appConfig;
     this.express.use(`/api/${version}`, routes);
+    this.express.use('/image', express.static('Upload/Images'));
```

Comment:
in the setup, we can create the upload/image directory if they don't exist.
Will do that in next PR